src/components/IconBase/IconBase.stories.jsx

```diff
@@ -16,5 +16,54 @@ export const Description = () => (
     <p>
       For examples of how this is used, see the other <code>Icon</code> stories.
     </p>
+    <h2>Accessibility</h2>
```

Comment:
Let's keep this in the [design.va.gov](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/543#pullrequestreview-767135897) documentation. Feel free to link to that page from here, though.

Reply:
I deleted this section and linked to the icon documentation.